src/main/java/frc/robot/Constants.java

```diff
@@ -5,6 +5,31 @@
 package frc.robot;
 
 public final class Constants {
+  
+    //Shooter
+    public static final int SHOOTER_MOTOR_ID_1 = 44;
+    public static final int SHOOTER_MOTOR_ID_2 = 45;
+
+    public static final int SHOOTER_SENSOR_ID_1 = 0;
+    public static final int SHOOTER_SENSOR_ID_2 = 1;
+
+    public static final double SHOOTER_MOTOR_SPEED = 0.75;
+
+    public static final double SHOOTER_MOTOR_1_RPM = 12000;
+    public static final double SHOOTER_MOTOR_2_RPM = 12000;
+
+    //Shooter motor PID constants
+    public static final double SHOOTER_MOTOR_PID_P = 5e-5;
+    public static final double SHOOTER_MOTOR_PID_I = 0;
+    public static final double SHOOTER_MOTOR_PID_D = 5e-5;
+    public static final double SHOOTER_MOTOR_PID_IZ = 0;
+    public static final double SHOOTER_MOTOR_PID_FF = 0.000015;
+    public static final double SHOOTER_MOTOR_MAX_OUTPUT = 1;
+    public static final double SHOOTER_MOTOR_MIN_OUTPUT = -1;
+    
+    //Controller
+    public static final int kDriverControllerPort = 2;
```

Comment:
We don't use the "wpilib" style of variable naming. Our constants should be all caps with underscores.